includes/definitions/quanta.yaml

```diff
@@ -4,11 +4,14 @@ type: network
 icon: quanta
 over:
     - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+mib_dir:
+    - quanta
 discovery:
     - sysObjectId:
           - .1.3.6.1.4.1.7244
-    -
-      sysObjectId:
```

Comment:
Actually I've just tested the change and it's the same for both so you can revert this (unless you really did find a reason to change it):

```
NEW
Array
(
    [discovery] => Array
        (
            [0] => Array
                (
                    [sysObjectId] => Array
                        (
                            [0] => .1.3.6.1.4.1.7244
                        )

                )

            [1] => Array
                (
                    [sysObjectId] => Array
                        (
                            [0] => .1.3.6.1.4.1.4413
                        )

                    [sysDescr_regex] => Array
                        (
                            [0] => /VxWorks/
                            [1] => /quanta/i
                        )

                )

        )

)
OLD:
Array
(
    [discovery] => Array
        (
            [0] => Array
                (
                    [sysObjectId] => Array
                        (
                            [0] => .1.3.6.1.4.1.7244
                        )

                )

            [1] => Array
                (
                    [sysObjectId] => Array
                        (
                            [0] => .1.3.6.1.4.1.4413
                        )

                    [sysDescr_regex] => Array
                        (
                            [0] => /VxWorks/
                            [1] => /quanta/i
                        )

                )

        )

)
```